samples/excel/82-document/get-file-in-slices-async.yaml

```diff
@@ -0,0 +1,201 @@
+                const sheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Sample");
+                let expensesTable = sheet.tables.add('A1:E1', true);
+                expensesTable.name = "SalesTable";
+
+                expensesTable.getHeaderRowRange().values = [["Product", "Qtr1", "Qtr2", "Qtr3", "Qtr4"]];
+
+                expensesTable.rows.add(null, [
+                    ["Frames", 5000, 7000, 6544, 4377],
+                    ["Saddles", 400, 323, 276, 651],
+                    ["Brake levers", 12000, 8766, 8456, 9812],
+                    ["Chains", 1550, 1088, 692, 853],
+                    ["Mirrors", 225, 600, 923, 544],
+                    ["Spokes", 6005, 7634, 4589, 8765]
+                ]);
+
+                if (Office.context.requirements.isSetSupported("ExcelApi", 1.7)) {
+                    sheet.getUsedRange().format.autofitColumns();
+                    sheet.getUsedRange().format.autofitRows();
+                }
+
+                sheet.activate();
+
+                await context.sync();
+                createChart();
+            });
+        } 
+
+        async function createChart() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const salesTable = sheet.tables.getItem("SalesTable");
+
+                const dataRange = salesTable.getDataBodyRange();
+
+                let chart = sheet.charts.add("ColumnClustered", dataRange, Excel.ChartSeriesBy.columns);
+
+                chart.setPosition("A15", "F30");
+                chart.title.text = "Quarterly sales chart";
+                chart.legend.position = "right"
+                chart.legend.format.fill.setSolidColor("white");
+                chart.dataLabels.format.font.size = 15;
+                chart.dataLabels.format.font.color = "black";
+                let points = chart.series.getItemAt(0).points;
+                points.getItemAt(0).format.fill.setSolidColor("pink");
+                points.getItemAt(1).format.fill.setSolidColor("indigo");
+
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            }
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+
+        declare namespace base64js {
+            /** Takes a byte array and returns a base64 string
+             * Imported from https://www.npmjs.com/package/base64-js package. */
+            function fromByteArray(array: number[]): string;
+        }
+    language: typescript
+template:
+    content: |
+        <section class="ms-font-m">
+            <p>This sample shows how to get a file in slices, and join the slices back again later.</p>
```

Comment:
Suggest that you change the description here to:

This sample shows how to get the base64-encoded string that represents the current document. It uses the getFileAsync() method to read the file in slices and then joins all slices back together to form the complete file.

Reply:
Thanks, done.